spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc

```diff
@@ -113,6 +125,34 @@ delayedExchange::
   The `x-delayed-type` argument is set to the `exchangeType`.
 +
 Default: `false`.
+dlqDeadLetterExchange::
+  if a DLQ is declared, a DLX to assign to that queue
```

Comment:
DLQ for DLQ?
If I haven't read a PR commit message, I wouldn't understand what are these all about:

> support DLX/DLQ configuration on the DLQ as well - combined with TTL can be used to route back to primary queue

Any ideas to improve these descriptions?

Thanks